tests/test_artifactory.py

```diff
@@ -29,18 +29,21 @@ def test_can_load_from_artifactory(open_estimator_pickle):
 
 
 @require_artifactory
-def test_can_save_to_artifactory(open_estimator_pickle, tmp_path: pathlib.Path):
+def test_can_save_to_artifactory(
+    open_estimator_pickle, tmp_path: pathlib.Path, regression
+):
     file_path = tmp_path.joinpath("temp.pkl")
 
     def mock_deploy_file(*args, **kwargs):
         return file_path.write_bytes(open_estimator_pickle.read())
 
     mock = MagicMock()
     mock.__truediv__.deploy_file.return_value = mock_deploy_file()
+    mock.is_file.return_value = False
```

Comment:
I would add some `mock.assert_called_` statements to check the internals as well